app/src/main/java/com/hammer/talkbbokki/presentation/comment/CommentDetailScreen.kt

```diff
@@ -0,0 +1,244 @@
+    CommentDetailScreen(
+        onBackClick,
+        recomments.value.first(),
+        recomments.value
+    )
+}
+
+@Composable
+fun CommentDetailScreen(
+    onBackClick: () -> Unit,
+    comment: Comment,
+    recomments: List<Comment>
+) {
+    Box(
+        Modifier
+            .fillMaxSize()
+            .background(MainBackgroundColor)
+    ) {
+        Column(
+            Modifier
+                .fillMaxSize()
+                .padding(bottom = 78.dp)
+        ) {
+            CommentDetailHeader(onBackClick)
+            CommentContents(
+                nickname = comment.nickname,
+                date = comment.date,
+                content = comment.content,
+                replyCount = comment.replyCount,
+                onDeleteClick = comment.onDeleteClick
+            )
+            RecommentList(
+                recomments
+            )
+        }
+        Box(Modifier.align(Alignment.BottomCenter)) {
+            RecommentInputArea()
+        }
+    }
+}
+
+@Composable
+fun CommentDetailHeader(
+    onBackClick: () -> Unit,
+) {
+    val replyCount = 0
+    Box(
+        Modifier
+            .height(56.dp)
+            .fillMaxWidth()
+    ) {
+        Icon(painter = painterResource(id = R.drawable.ic_arrow_left),
+            tint = White,
+            contentDescription = null,
+            modifier = Modifier
+                .clickable {
+                    onBackClick()
+                }
+                .align(Alignment.CenterStart)
+                .padding(start = 20.dp))
+        Text(
+            text = "답글 ($replyCount)", Modifier.align(Alignment.Center), color = White
+        )
+    }
+}
+
+@Composable
+fun RecommentItem(
+    nickname: String, date: String, content: String, replyCount: Int, onDeleteClick: () -> Unit
+) {
+    Box(modifier = Modifier.padding(20.dp)) {
+        Image(
+            modifier = Modifier.size(18.dp),
+            painter = painterResource(id = R.drawable.ic_recomment),
+            contentDescription = null
+        )
+        CommentContents(
+            nickname = nickname,
+            date = date,
+            content = content,
+            replyCount = replyCount,
+            onDeleteClick = onDeleteClick
+        )
+    }
+}
+
+@Composable
+fun CommentContents(
+    nickname: String, date: String, content: String, replyCount: Int, onDeleteClick: () -> Unit
+) {
+    val isMine = false
+    val isParentComment = false
+
+    Column(modifier = Modifier.padding(start = 26.dp)) {
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            Text(
+                text = nickname, style = TalkbbokkiTypography.b3_bold,
+                color = if (isMine) MainColor01 else White
+            )
+            Spacer(modifier = Modifier.width(8.dp))
+            Text(text = date, style = TalkbbokkiTypography.caption, color = Gray06)
+        }
+        if (isMine) {
+            Icon(painter = painterResource(id = R.drawable.ic_close),
```

Comment:
🚫 **[ktlint]** <standard:argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)